aiida_mlip/parsers/opt_parser.py

```diff
@@ -0,0 +1,119 @@
+"""
+Geom optimisation parser.
+"""
+
+from pathlib import Path
+from typing import Union
+
+from ase.io import read
+
+from aiida.common import exceptions
+from aiida.engine import ExitCode
+from aiida.orm import SinglefileData, StructureData, TrajectoryData
+from aiida.orm.nodes.process.process import ProcessNode
+from aiida.plugins import CalculationFactory
+
+from aiida_mlip.parsers.sp_parser import SPParser
+
+geomoptCalculation = CalculationFactory("janus.opt")
+
+
+def xyz_to_aiida_traj(traj_file: Union[str, Path]):
```

Comment:
It doesn't have to be moved now, but I'm guessing this will probably end up being in a utils module, since it'll probably be useful for MD etc too.

Reply:
yap let us cross that bridge when we come to it... Friday I hope.